# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../binutils/have_stdlib.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./libiberty/cplus-dem.c.orig	Thu Nov  8 01:11:59 2001
+++ ./libiberty/cplus-dem.c	Thu Nov  8 01:14:09 2001
@@ -48,12 +48,7 @@
 #pragma weak dlerror
 #endif
 
-#ifdef HAVE_STDLIB_H
 #include <stdlib.h>
-#else
-char * malloc ();
-char * realloc ();
-#endif
 
 #include <demangle.h>
 #undef CURRENT_DEMANGLING_STYLE
